cartridge/roles/expirationd.lua

```diff
@@ -137,6 +137,33 @@ local function get_task_config(task_conf)
     return conf
 end
 
+local function get_cfg(cfg)
```

Comment:
So in fact "get" is "validate and fill with defaults?"

Reply:
What would happen if one uploads
```yml
expirationd:
   cfg:
```
? Will defaults override existing configuration?

Reply:
> What would happen if one uploads
> 
> ```yaml
> expirationd:
>    cfg:
> ```
> 
> ? Will defaults override existing configuration?

It will do nothing.

Reply:
> > What would happen if one uploads
> > ```yaml
> > expirationd:
> >    cfg:
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > ? Will defaults override existing configuration?
> 
> It will do nothing.

I get it now. Maybe we should add a simple test:
- set non-default value,
- upload `{expiractiond = cfg = {}}`,
- assert non-default value is still here.

Reply:
Added:
```test_apply_config_cfg_empty_do_nothing```